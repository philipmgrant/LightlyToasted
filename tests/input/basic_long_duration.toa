|Duration: LONG
testing1
testing2
testing3