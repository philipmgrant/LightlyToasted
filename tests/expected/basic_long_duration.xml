<toast duration="long"><visual><binding template="ToastImageAndText04"><image id="1" src=""/><text id="1">testing1</text><text id="2">testing2</text><text id="3">testing3</text></binding></visual></toast>